tsk/fs/fs_attr.c

```diff
@@ -209,12 +210,13 @@ fs_attr_put_name(TSK_FS_ATTR * fs_attr, const char *name)
  * @param res_data Pointer to where resident data is located (data will
  * be copied from here into FS_DATA)
  * @param len Length of resident data
+ * @param offset Offset on disk
```

Comment:
It looks like -1 is being used to mean special things. We should document that in this API and in the rd struct.

Reply:
Okay. Let me know how you want to do that?